eo-maven-plugin/src/test/java/org/eolang/maven/DiscoverMojoTest.java

```diff
@@ -174,6 +177,52 @@ void discoversWithSeveralObjectsWithDifferentVersions(
         );
     }
 
+    @Test
+    void discoversDifferentUnversionedObjectsFromDifferentVersionedObjects(@TempDir final Path tmp)
+        throws IOException {
+        final Map<String, CommitHash> hashes = new CommitHashesMap.Fake();
+        final String program = String.join("\n", "[] > sprintf", "  \"Hello world\" > @");
+        final String one = hashes.get("0.28.1").value();
+        final String two = hashes.get("0.28.2").value();
+        final String string = "org.eolang.string";
+        final String sprintf = "foo/x/sprintf";
+        final String object = "foo.x.sprintf";
+        final String ext = "%s.eo";
+        final String format = String.join("_", "%s", ext);
+        final ForeignTojos tojos = new FakeMaven(tmp)
+            .with("withVersions", true)
+            .withProgram(
+                Paths.get(String.format(format, sprintf, one)),
+                program,
+                new OnVersioned(object, one)
+            )
+            .withProgram(
+                Paths.get(String.format(format, sprintf, two)),
```

Comment:
@maxonfjvipon Why do we need to create these paths? Why don't we just delegate it to `FakeMaven` (as it was previously)?